### PR TITLE
Ical latest

### PR DESCRIPTION
**Pull request by matthiasbeyer**
This PR contains some stuff:
- Support to build a ical-latest file, which includes only the last N events
- Support to configure the site with the `nanoc.yaml` file, not by editing source
- Some code cleanup for the ical generator code

@hfuunfug/members  feel free to review.

### CODE REVIEW

File: lib/ical_helpers.rb [hunk 1]
```diff
@@ -10,13 +19,21 @@
...
-    ical.add_event event
+    ical.add_event generate_event(item, cfg)
   end
 
   @items << Nanoc::Item.new(ical.to_ical, {:kind => :ical}, "/ical/")
 end
+
+def generate_ical_items_latest cfg
+  latest = cfg[:page][:ical][:latest]
+  return if latest.nil? or latest == false
```
Comment (TheNeikos) [inline lib/ical_helpers.rb:30]: Why would it be false?

Comment (matthiasbeyer) [inline lib/ical_helpers.rb:30]: You can set the config to `false` to disable the generating of the ical file with the latest entries.

Comment (TheNeikos) [inline lib/ical_helpers.rb:30]: Alright

**Comment by TheNeikos** [comment]
I see nothing wrong with the code, can't run it right now, but I suppose you tested that already?

**Comment by matthiasbeyer** [comment]
Yes, I tested it. Feel free to test anyways.

**Pull request merged**